examples/stable/test/property/ic_api/call/test/tests.ts

```diff
@@ -0,0 +1,28 @@
+import { describe } from '@jest/globals';
+import { please, Test } from 'azle/_internal/test';
+import { execSync } from 'child_process';
+
+export function getTests(): Test {
+    return () => {
+        describe.each([
```

Comment:
Please remove this `describe.each` and the comment below, `getTests` should just return the empty lambda.